Blob.js

```diff
@@ -14,12 +14,24 @@
 
 /*! @source http://purl.eligrey.com/github/Blob.js/blob/master/Blob.js */
 
-(function (view) {
+(function(global) {
+(function (factory) {
+  if (typeof define === "function" && define.amd) {
+    // AMD. Register as an anonymous module.
+    define(["exports"], factory);
```

Comment:
Could you match the existing whitespace usage in the file? It's currently using tabs.